test/samples/if_assigned_complex.instrumented.sol

```diff
@@ -60,83 +60,101 @@ contract Base {
 
     function Base_x_inline_initializer() internal {
         vars1 memory _v;
-        _v.old_0 = x;
-        if (!(x >= _v.old_0)) {
-            emit AssertionFailed("0: ");
-            assert(false);
+        unchecked {
+            _v.old_0 = x;
+        }
```

Comment:
It looks a bit strange, because I do not think if overflow/underflow may happen with such plain assignments. Is this a side effect from some changes in the code base?

Reply:
This is a side effect of me just wrapping everything related to property computation in `unchecked` blocks, regardless of whether it contains arithmetic. Its just easier to write it that way.